tests/Client/GetUserGroupsClientTest.php

```diff
@@ -322,6 +326,8 @@ public function testGetUserProducesCorrectInputForEmployeeID() {
         self::assertCount(1, $container);
         $request = $container[0]['request'];
         $body = (string) $request->getBody();
+        $body = urldecode($body);
+        $body = str_replace('Package=', '', $body);
```

Comment:
If the string `Package=` were to be in one of the request fields it would also be removed as written.

```suggestion
        $decodedBody = urldecode((string) $request->getBody());
        $body = strrpos($decodedBody, 'Package=') === 0 ? substr($decodedBody, 8, null) : '';
```